lib/Runtime/Library/TypedArray.cpp

```diff
@@ -175,6 +171,11 @@ namespace Js
                             Var lengthVar = JavascriptOperators::OP_GetProperty(jsArraySource, PropertyIds::length, scriptContext);
                             if (JavascriptOperators::GetTypeId(lengthVar) == TypeIds_Undefined)
```

Comment:
Updated PR to use original codepath order. Gain perf by keeping arrays out of the slower codepath for iterables.